fission/main.go

```diff
@@ -17,11 +17,105 @@ limitations under the License.
 package main
 
 import (
+	"errors"
+	"fmt"
 	"os"
+	"path/filepath"
 
+	"github.com/skratchdot/open-golang/open"
 	"github.com/urfave/cli"
+	"k8s.io/client-go/1.5/kubernetes"
```

Comment:
Would you mind splitting the k8s-client stuff into a separate file in this package?